templates/web/base/waste/confirmation.html

```diff
@@ -15,9 +19,16 @@ <h1 class="govuk-panel__title">
         [% title %]
     </h1>
     <div class="govuk-panel__body">
-        <p>Your [% title | lower %];
-          [% IF report.user.email && report.get_extra_metadata('contributed_as') != 'anonymous_user' %]
-            a copy has been sent to your email address, [% report.user.email %].
+        <p>
+          [% IF c.cobrand.moniker == 'bexley' %]
```

Comment:
Now I'm looking at this page, this (not your change, the actual page) seems a bit odd! Showing the title, and then showing it again with "Your" at the start. Doesn't look great (presumably why Bexley asked for it to be changed, in part). So I'd be happy if we changed the titles to be what Bexley suggested for all missed collections, did similar for the other titles, didn't repeat the title with "Your", so basically having what Bexley have here everywhere, if that makes sense, it does seem better overall. Dunno how many test would need updating, hopefully not many.)